package.json

```diff
@@ -27,18 +30,20 @@
   },
   "homepage": "https://github.com/ciena-frost/frost-standard#readme",
   "devDependencies": {
-    "chai": "^3.4.1",
-    "eslint": "^3.0.1",
+    "chai": "^4.0.0",
+    "eslint": "^4.1.0",
     "istanbul": "^0.4.2",
-    "mocha": "^2.4.2"
+    "mocha": "^3.4.2"
   },
   "dependencies": {
-    "babel-eslint": "^6.1.2",
-    "eslint-config-standard": "^5.3.5",
+    "babel-eslint": "^7.2.3",
+    "eslint-config-standard": "^10.2.1",
     "eslint-plugin-ember-standard": "0.0.23",
-    "eslint-plugin-mocha": "^4.7.0",
+    "eslint-plugin-import": "^2.2.0",
```

Comment:
Would prefer this to not be a float so that unexpected rules do not suddenly appear in   the codebase and fail unexpectedly.  This particular repo seems to disable new rules by default but even so that's a lot of assumptions.